dbms/src/Storages/Page/V2/gc/LegacyCompactor.cpp

```diff
@@ -290,7 +290,7 @@ size_t LegacyCompactor::writeToCheckpoint(const String & storage_path,
         auto checkpoint_writer = checkpoint_file.createWriter(false, true);
 
         PageEntriesEdit edit;
-        bytes_written += checkpoint_writer->write(wb, edit, write_limiter);
+        bytes_written += checkpoint_writer->write(wb, edit, write_limiter, /*background*/ true);
```

Comment:
Should also mark the read flow in `PageFile::MetaMergingReader::initialize` as background ...
https://github.com/pingcap/tiflash/blob/090e34f34b50ecd4b10607345fb3e86a1cd67cc4/dbms/src/Storages/Page/V2/PageFile.cpp#L525-L527

so that we can see the read flow for these codes as "background"

https://github.com/pingcap/tiflash/blob/090e34f34b50ecd4b10607345fb3e86a1cd67cc4/dbms/src/Storages/Page/V2/gc/LegacyCompactor.cpp#L134-L155

Reply:
I see, but I am thinking that should we only add `PageBackground` in the `data` part?

Because meta part GC flow is not too much. And if I do this, Then we can't distinguish the traffic of `meta` and `data`?

Reply:
The read flow for meta part could be large under some situations. For example, there are many "legacy" PageFile left on the disk, but GC can not push forward because some snapshots are being hold for a long time. However, each round in V2 GC, we need to scan all meta from the disk. 
We want to distinguish the traffic for foreground R/W and GC R/W but not the meta and data.

Reply:
okay,added.